core-requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy>=1.20.0
+numpy>=1.21.1
```

Comment:
Do we need to bump the numpy version for this? I assume the list conversion would work regardless of whether its `1.20.0` or `1.21.1`